msg/ObjectHypothesis.msg

```diff
@@ -5,7 +5,7 @@
 # The unique ID of the object class. To get additional information about
 #   this ID, such as its human-readable class name, listeners should perform a
 #   lookup in a metadata database. See vision_msgs/VisionInfo.msg for more detail.
-string id
+string class_id
```

Comment:
I really think you should just leave this alone, I think its fine as is

Reply:
Sorry, I disagree. I always found it extremely weird that this field is called `id`. I think this is because a Detection is about an object, so you have things like (shortened) `object.pose`, `object.bbox` and then `object.id = "mug"`, except it's not the object's id, but its class. `object.class` would be nice, but that name is already taken by Java, hence `class_id`. `type` or `classification` would also work for me, but I think `class_id` captures the meaning best.

Reply:
To be more precise, it would read `hypothesis.id`, but it's not the id of the hypothesis, but the class.

Reply:
@Kukanani thoughts?

Reply:
I personally like `class_id`. @SteveMacenski do you see potential downsides to changing it to this new name, other than breaking backwards compatibility / requiring migration?

I also think that renaming `tracking_id` to `instance_id` is more important and should take place regardless of whether or not we also do this rename.

Reply:
Lets try to keep these 2 changes separate, I don't want to lump them together since they mean different things. 

`class_id` is fine with me